server/lib/schema/index.ts

```diff
@@ -2158,6 +2165,10 @@ const schema = new GraphQLSchema({
                       isPublic: false,
                       shouldOverrideToNormalUser: false,
                     });
+
+                    await judgeChild.update({
+                      isStale: maliciousAnswer && honestAnswer ? true : false,
```

Comment:
If a judge workspace is created using this GraphQL mutation, it will always be the leaf workspace, and so should always be stale. Another way to get a feel for what is going on is notice that this line is within the scope of two conditionals: the outer conditional guaranteeing `honestAnswer` has been provided and the inner one guaranteeing `maliciousAnswer` has been provided, so `maliciousAnswer && honestAnswer` will always be true.

Reply:
Good catch.

It seems Mosaic automatically does workspace creation / staleness adjustment outside this function. I'm trying to figure out where that happens, as the above logic doesn't result in the behavior that I'd expect.

Also planning to set up a CSV file that is structured to test all the cases we enumerated.

Reply:
Mosaic, when creating a workspace, always sets `isStale` to true. It's true though that there is a lot of logic, spread throughout the app, that can then set `isStale` to false (indicating that the workspace shouldn't be scheduled again unless something significant happens, like all of its subquestions have answers) or set it to true again (indicating the workspace should be scheduled again, all else equal). 

None of this complexity is that relevant here though. This API is used to do the first few workspaces of a tree ahead of time, with some flexibility as to exactly how many to do. We need to make sure that the leaf workspace, whichever it is, is eligible to get scheduled to actual experiment participants (i.e., is marked as stale) and the non-leaf workspaces aren't eligible (since they've already been handled by the user of the API).

Reply:
But yeah it is important to know that the default value for `isStale` is true. You can see this in the workspace model.

So for this PR we just need to set `isStale` as false for any non-leaf workspace.